client/src/components/AuditAdmin/useRoundsAuditAdmin.ts

```diff
@@ -65,57 +43,59 @@ export const drawSampleError = (rounds: IRound[]): string | null =>
 export const isAuditStarted = (rounds: IRound[]): boolean =>
   rounds.length > 0 && isDrawSampleComplete(rounds) && !drawSampleError(rounds)
 
-const useRoundsAuditAdmin = (
+export const useRounds = (
   electionId: string,
-  refreshId?: string
-): [
-  IRound[] | null,
-  (sampleSizes: ISampleSizes) => Promise<boolean>,
-  () => Promise<boolean>
-] => {
-  const [rounds, setRounds] = useState<IRound[] | null>(null)
+  options?: UseQueryOptions<IRound[], ApiError, IRound[], string[]>
+): UseQueryResult<IRound[], ApiError> =>
+  useQuery(
+    ['elections', electionId, 'rounds'],
```

Comment:
I don't feel strongly about this, but could move the construction of this key to a function so that it can be reused in the invalidation calls below, too

I could see someone accidentally updating this key while forgetting to update the keys below

Reply:
Good point. I'll do that. I'd like to figure out an approach that can work for all our queries, since they all have keys...